misk-crypto/src/main/kotlin/misk/crypto/internal/Aead.kt

```diff
@@ -0,0 +1,26 @@
+package misk.crypto.internal
+
+import com.google.crypto.tink.Aead
+
+/**
+ * Decorates {@link com.google.crypto.tink.Aead} to collect metrics on operations.
+ */
+class Aead(
+  private val key: misk.crypto.Key,
+  private val aead: com.google.crypto.tink.Aead,
+  private val metrics: KeyMetrics,
+) : Aead by aead {
+
+  override fun encrypt(plaintext: ByteArray?, associatedData: ByteArray?): ByteArray {
```

Comment:
Please change `plaintext: ByteArray?` to `plaintext: ByteArray`. That parameter **can't** be null (see the original Java interface in the Tink library).

This applies to all the rest of the decorated extended primitive classes here.